### DISCUSSION TITLE
ChatGPT suggests strange version of this library

### DISCUSSION THREAD

**Discussion by Tardigrada777** [freesewing/freesewing#3462] [Support] [answered]
What is the version of freesewing library ChatGPT suggests in the next examples?

```js
const freesewing = require("freesewing");
const { Pattern, measurements } = freesewing;

const shirt = new Pattern();

shirt.measurement("cm");
shirt.size(34, 54);

// Define your measurements here
shirt.measure(measurements.a, X);
shirt.measure(measurements.b, Y);
shirt.measure(measurements.c, Z);
shirt.measure(measurements.e, W);

// Placeholder for the draft function
shirt.draft(() => {
  // Placeholder for creating paths and points
});

// Placeholder for any additional functions

// Generate the pattern
shirt.render("pdf", "shirt");
```

I can't find right version from the versions history on GitHub.

**Comment by joostdecock** [comment, marked as answer]
There is no package `freesewing`, or rather there hasn't been for years.
The first line should load the core library:

```js
const freesewing = require("@freesewing/core")
```

But there's going to be other problems with this code as well.
For one thing, we'll be migrating to v3 soon which is pure ESM, and this is CJS code.

If you'd like to learn, we have developer docs at https://freesewing.dev/
If you'd like to play around with ChatGPT that's fine too. But perhaps don't rely on community support for it.

**Comment by Tardigrada777** [comment]
Thank you for answer! I just wanted to find the version which chatGPT uses, but that seems impossible and community has no idea on that. And even docs can't help me. Hope that on day we will see the well organized releases history.

**Comment by joostdecock** [comment]
Closing to give room to active discussions
